apis/r/R/ephemeral.R

```diff
@@ -0,0 +1,450 @@
+#' Ephemeral Collection Base
+#'
+#' @description Base class for ephemeral collections
+#'
+#' @keywords internal
+#'
+#' @export
+#'
+EphemeralCollectionBase <- R6::R6Class(
+  classname = 'EphemeralCollectionBase',
+  inherit = SOMACollectionBase,
+  public = list(
+    # Overwrite TileDBObject methods
+    #' @description Create an ephemeral collection
+    #'
+    #' @template param-dots-ignored
+    #'
+    #'
+    initialize = function(...) {
+      if (rlang::dots_n(...)) {
```

Comment:
tiledb and czi both are _highly_ polyglot companies -- any cleverness whatsoever (be it Python or R or anything else) -- i would ask for a quick comment on what this does

something like `os.path_exists(foo)` is clear without a comment; `if (rlange::dots_n(...))` is (to me) not